src/main/java/org/zalando/nakadi/repository/db/SchemaRepository.java

```diff
@@ -28,6 +34,29 @@ public SchemaRepository(final JdbcTemplate jdbcTemplate, final ObjectMapper obje
                 new SchemaRowMapper());
     }
 
+    public EventTypeSchema getSchemaVersion(final String name, final String version)
```

Comment:
Good point. Fixed in https://github.com/zalando/nakadi/pull/497/commits/c88ad820765bc55cf1e7a79af81be1bfffd02c17